samples/server/petstore/cpp-qt5-server/server/src/models/OAIHelpers.cpp

```diff
@@ -0,0 +1,704 @@
+/**
+ * OpenAPI Petstore
+ * This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
+ *
+ * OpenAPI spec version: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+#include "OAIHelpers.h"
+#include "OAIModelFactory.h"
+#include "OAIObject.h"
+#include <QDebug>
+#include <QJsonArray>
+#include <QJsonValue>
+
+
+
+namespace OpenAPI {
+
+void
+setValue(void* value, QJsonValue obj, QString type, QString complexType) {
+    if((value == nullptr) || (obj.isUndefined())) {
+        // can't set value with a null pointer
+        return;
+    }
+    if(QStringLiteral("bool").compare(type) == 0) {
+        bool * val = static_cast<bool*>(value);
```

Comment:
I think you can use qvariant_cast and QVariant here to avoid storing pointers

Reply:
I tried it and sadly the model became complicated :|